### ISSUE TITLE
[Documentation][Configuration] Missing/incomplete documentation about configuration and/or missing configuration button

### ISSUE THREAD

**Issue by prof7bit** [MTrab/landroid_cloud#278]
The README states that the yaml configuration is "legacy" but it does not mention how to configure it otherwise.

I have been using this integration for a few months now, it has been mostly working. Yesterday I had some connectivity problems that seemed to have resolved themselves somehow on their own later the same day, but while I was investigating I noticed that I am not able to view or change the login credentials for this integration.

I cannot remember how exactly I configured it initially, but it does NOT seem like I have used the legacy method, I cannot find any Landroid related config entries in my configuration.yaml, or anywhere else. There must have been a dialog to enter the credentials, but I don't seem to be able to enter this dialog again. Some other integrations offer a "configure" button where you can enter credentials, but this integration doesn't.

Should I ever need to change the login credentials I would have absolutely no idea how to do this, other than maybe uninstalling and reinstalling and then hoping to somehow stumble over the initial configuration dialog again.

I'm not sure whether this is a documentation issue or whether I am missing something fundamental about home assistant, or whether this is really a bug and the configuration button should be visible (I think it should).

**Comment by MTrab** [comment]
It's configured in integrations UI, as most others are these days - hence the legacy note for YAML (might even remove the YAML option in the future as more and more config moves to the UI)

**Comment by prof7bit** [comment]
Then it is a bug, because there is no way to configure it in the integrations UI. Please reopen.

**Comment by prof7bit** [comment]
See here: its missing:
![no-config-button](https://user-images.githubusercontent.com/1401589/196659083-c9b6a44f-a821-479a-8350-7b558e36443f.png)

**Comment by MTrab** [comment]
What do you want to reconfigure?

**Comment by prof7bit** [comment]
The login credentials. I once somehow(?) configured them after initial installation, but now there is no dialog to changes this data anymore and the configuration via yaml is marked as "legacy" and it is not documented whether these "legacy" settings would have precedence over the new ones (that are inaccessible now).

You wrote that the credentials can be edited via integrations UI.

This bug is about the fact that this is obviously not possible because there simply **IS** no way to edit any kind of setting at all via the UI (the button is missing) and it is also not documented where the credentials are stored so they could at least be edited with a text editor.

**Comment by MTrab** [comment]
No, didn't write it could be changed. But all config is thru the UI.
To change credentials you need to remove the current instance of the integration and re-add it - all via the integration page, just like any other integration.
Credentials always (well as a rule of thumb at least) requires to remove and add.

So no, no big or anything missing.
